integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -98,29 +104,59 @@ public Network getNetwork() {
   }
 
   public void startHiveContainer() {
-    startHiveContainer(
-        ImmutableMap.<String, String>builder().put("HADOOP_USER_NAME", "anonymous").build());
+    if (hiveContainer == null) {
+      synchronized (ContainerSuite.class) {
+        if (hiveContainer == null) {
+          // Start Hive container
+          HiveContainer.Builder hiveBuilder =
+              HiveContainer.builder()
+                  .withHostName("gravitino-ci-hive")
+                  .withEnvVars(
+                      ImmutableMap.<String, String>builder()
+                          .put("HADOOP_USER_NAME", "anonymous")
+                          .build())
+                  .withNetwork(network);
+          HiveContainer container = closer.register(hiveBuilder.build());
+          container.start();
+          hiveContainer = container;
+        }
+      }
+    }
   }
 
   /**
-   * To start the Hive container, you can to specify environment variables: HIVE_RUNTIME_VERSION:
-   * Hive version, currently support `hive2`(default) and `hive3` DOCKER_ENV_RANGER_SERVER_URL:
-   * Ranger server URL DOCKER_ENV_RANGER_HIVE_REPOSITORY_NAME: Ranger Hive repository name
-   * DOCKER_ENV_RANGER_HDFS_REPOSITORY_NAME: Ranger HDFS repository name
+   * To start and enable Ranger plugin in Hive container, <br>
+   * you can specify environment variables: <br>
+   * 1. HIVE_RUNTIME_VERSION: Hive version, currently support `hive2`(default) and `hive3` <br>
```

Comment:
I believe you need to clarify why we support `hive2` and `hive3` in the Java document but only support `hive3` in the code, see line 141.

Reply:
I already update document in the here: https://github.com/apache/gravitino/blob/main/docs/docker-image-details.md#apache-gravitino-ci-apache-hive-image (gravitino-ci-hive:0.1.13)

Reply:
I mean those comments are for the container `hiveRangerContainer` NOT for the image,  I got that the image supports both `hive2` and `hive3`.

Reply:
DONE.